session/bootstrap.go

```diff
@@ -1942,6 +1946,13 @@ func upgradeToVer95(s Session, ver int64) {
 	doReentrantDDL(s, "ALTER TABLE mysql.user ADD COLUMN IF NOT EXISTS `User_attributes` JSON")
 }
 
+func upgradeToVer96(s Session, ver int64) {
+	if ver >= version96 {
+		return
+	}
+	doReentrantDDL(s, "ALTER TABLE mysql.user ADD COLUMN IF NOT EXISTS `Token_issuer` varchar(255)")
```

Comment:
You may also need to add the field in  https://github.com/pingcap/tidb/blob/master/session/bootstrap.go#L62